daemon/api_validate.go

```diff
@@ -141,7 +143,29 @@ func listValidationSets(c *Command, r *http.Request, _ *auth.UserState) Response
 	return SyncResponse(results, nil)
 }
 
-func getValidationSet(c *Command, r *http.Request, _ *auth.UserState) Response {
+var checkInstalledSnaps = func(vsets *snapasserts.ValidationSets, snaps []*snapasserts.InstalledSnap) error {
+	return vsets.CheckInstalledSnaps(snaps)
```

Comment:
this is never tested, also given it has no side-effects , I suppose we could call it wrap it to do extra checks but still call it for real as well?  

we use that pattern of wrap but call in some other places in the daemon tests I think

Reply:
Done.